### PR TITLE
Rework API creation Get Started page

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/7645

**Description**

add new screen for api creation get started
- add new screen for creation get started links
- keep legacy screen only for creation by import

**Additional context**
image

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/7645-promite-api-designer/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-nvocoquozl.chromatic.com)

### CODE REVIEW

**Review by gaetanmaisse — APPROVED** [review]
✅  My only feedback will be on the images: can you install [ImageOptim](https://imageoptim.com/mac): `brew install --cask imageoptim` and use it to optimize the 2 images you are adding?

**Review by gaetanmaisse — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/creation/api-creation-get-started.component.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  ngOnDestroy() {
+    this.unsubscribe$.next(true);
+    this.unsubscribe$.unsubscribe();
+  }
+
+  private computeCockpitLink(installation: Installation) {
+    if (installation?.additionalInformation.COCKPIT_INSTALLATION_STATUS === 'ACCEPTED') {
+      this.cockpitLink = installation.cockpitURL;
+    } else {
+      this.cockpitLink = 'https://www.gravitee.io/platform/api-designer';
```
Comment (mouligno) [inline gravitee-apim-console-webui/src/management/api/creation/api-creation-get-started.component.ts]:
```suggestion
      this.cockpitLink = 'https://www.gravitee.io/platform/api-designer?utm_source=apim';
```

Comment (mouligno) [reply gravitee-apim-console-webui/src/management/api/creation/api-creation-get-started.component.ts]: Would be valuable for marketing and analytics :)

**Comment by ThibaudAV** [comment]
@gaetanmaisse I did it but it tells me 0% I think they are already optimized as I got them from gravitee.io

File: gravitee-apim-console-webui/src/management/api/creation/api-creation-get-started.component.ts (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/creation/api-creation-get-started.component.ts]: done

**Pull request merged**